projects/common-components/src/lib/buttons/submit/submit.component.html

```diff
@@ -0,0 +1,3 @@
+<button class="btn">
```

Comment:
Please, use jsmu-button instead of creation separated button.
All buttons should be implemented as jsmu-button, you can add modificator according to BEM.